simplejira/prompt.py

```diff
@@ -222,6 +222,7 @@ def __init__(self, jira_wrapper, issue):
             'do_10': 'do_pull',
             'do_11': 'do_remove',
             'do_11': 'do_exit',
+            'do_12': 'do_assign',
```

Comment:
This is picky but I think exit should be last in the printed list of cmds -- so can we just move assign up to #11 and exit as #12

Reply:
I was thinking of that, but thought let you say it so and then I'll update the sequence. Not picky.